modules/admission-webhook-about.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-about_{context}"]
+= Admission webhooks
+
+In addition to {product-title} default admission controller plug-ins, admission webhooks enable you to introduce admission plug-ins invoked from webhook servers over HTTP at defined endpoints.
```

Comment:
Why "*from* webhook servers"?

Reply:
This has now been reworded as per our conversation today.